INSTALL.sh

```diff
@@ -17,9 +17,9 @@ echo "This script will install PESTO on your system, would you like to proceed?
 echo "                                                                               "
 tput sgr0
 
-read CONFIRM
+read -r CONFIRM
 
-if [[ "$CONFIRM" != 'y' && "$CONFIRM" != 'Y' && "$CONFIRM" = "" ]]; then
+if [[ "$CONFIRM" != 'y' && "$CONFIRM" != 'Y' && "$CONFIRM" != "" ]]; then
```

Comment:
Last check was intended to be "=" because if the user presses return without reading, the installation must abort